test/test_ihate.py

```diff
@@ -43,6 +44,17 @@ def test_hate(self):
                          "artist:testartist album:notthis"]
         self.assertTrue(IHatePlugin.do_i_hate_this(task, match_pattern))
 
+    def test_summary(self):
+
+        # Task is not album
+        test_item = Item(
+            artist='TestArtist',
+            title='TestTitle')
+        task = importer.SingletonImportTask(None, test_item)
+        self.assertEqual(
+            summary(task),
+            'TestArtist - TestTitle')
+
```

Comment:
I'm not convinced that this is a useful test: It doesn't test any of the plugin's business logic, and the tested function is really very straightforward. The test only asserts that we're not inadvertently modifying the UI. However, this is not machine-readable output, and doesn't come with any stability guarantees in the first place. So, it'd say the only real result here is to make it more difficult for anybody who intentionally wants to modify the plugin's UI.

What was you motivation to add it?